File: easy/Sum-Of-Digits/Sum-Of-Digits.js
var fs  = require("fs");
fs.readFileSync(process.argv[2]).toString().split('\n').forEach(function (line) {
    if (line != "") {
        result = 0;
        for(i=0; i<line.length;i++){
            result += parseInt(line.substring(i,i+1));
        }
        console.log(result);
    }
});

File: easy/Sum-Of-Digits/Sum Of Digits.java
import java.io.*;
public class Main {
    public static void main(String[] args) throws FileNotFoundException, IOException {
        // TODO code application logic here
        File file = new File(args[0]);
        BufferedReader buffer = new BufferedReader(new FileReader(file));
        String line;
        while ((line = buffer.readLine()) != null) {
            line = line.trim();
            sum(line);
        }
    }

    private static void sum(String line) {
        int result = 0;
        for( int i=0; i<line.length();i++){
            result += Integer.valueOf(line.substring(i,i+1));
        }
        System.out.println(result);
    }
}
